source/features/show-names.tsx

```diff
@@ -27,7 +25,8 @@ function appendName(element: HTMLAnchorElement, fullName: string): void {
 	const insertionPoint = parentElement!.tagName === 'STRONG' ? parentElement! : element;
 	const nameElement = (
 		<span className="color-fg-muted css-truncate d-inline-block ml-1">
-			(<bdo className="css-truncate-target" style={{maxWidth: '200px'}}>{fullName}</bdo>)
+			{/* .css-truncate-target sets display: inline-block and can confine bidi overrides */}
+			(<span className="css-truncate-target" style={{maxWidth: '200px'}}>{fullName}</span>)
```

Comment:
We should fetch the original buggy test URL from a few years ago and add it to the list below.

Reply:
😅 Oh you are right, I just noticed we also merge names when appropriate, and that case is not fixed yet

Reply:
I think you're referring to when we change "georgemichael" to "George Michael" instead of adding a new name next to it. But I don’t think that can ever happen with RTL scripts

Reply:
It's not RTL, it's that bidi override can still break stuff in the merged case (the if branch in following code, this diff is the `appendName` in the else branch)

https://github.com/refined-github/refined-github/blob/76d23a0d6d78f54a34f4e956bf58f8733034f4c0/source/features/show-names.tsx#L68-L72

Reply:
Interestingly, it does (why is `isUsernameAlreadyFullName` so broad?) but I'm not seeing that bug in v24.12.20 (Safari) so I think there's nothing to fix (i.e. no need to add `d-inline-block`)